reana/reana_dev/utils.py

```diff
@@ -335,3 +337,93 @@ def get_prefixed_component_name(component):
     :return: Prefixed name.
     """
     return "-".join([INSTANCE_NAME, component])
+
+
+def fetch_latest_pypi_version(package):
+    """Fetch latest released version of a package."""
+    import requests
+
+    pypi_rc_info = requests.get("https://pypi.python.org/pypi/{}/json".format(package))
+    return sorted(pypi_rc_info.json()["releases"].keys())[-1]
+
+
+def update_module_in_cluster_components(
+    module,
+    new_version,
+    components_to_update=None,
+    use_latest_known_tag=True,
+    force_feature_branch=False,
+):
+    """Update the specified module version in all affected components."""
+
+    def _check_is_feature_branch(component):
+        branch = get_current_branch(get_srcdir(component))
+        if branch == "master":
+            click.secho(
+                f"[ERROR] Component {component} current branch is master.",
+                err=True,
+                fg="red",
+            ),
+            sys.exit(1)
+
+    def _replace_pypi_version(
```

Comment:
As you mentioned in the chat, maybe we can come up with something generic for this find and replace in different files? There is a [different implementation in `git-create-release-commit`](https://github.com/reanahub/reana/pull/372/files#diff-5ac538e6b7a8256fc232e7322372c8caR213) for a slightly different use case. Either we do it now, or we merge as is and refactor _a posteriori_, what do you think?

Reply:
I'll give it a try now and I'll come back to you.

Reply:
You can check it now. I've tried to be as explicit as possible.